docs/api-changes.md

```diff
@@ -63,6 +67,15 @@ Use the new `when` clause under a workflow, you can use the value of a boolean p
 ### GET openapi.json or GET openapi.yml
 Gives you the current production OpenAPI spec for the v2 API (eg: <https://circleci.com/api/v2/openapi.json> )
 
+### GET /insights/:project-slug/workflows/:workflow-name?branch=``
```

Comment:
Do we want to include a reference to the aggregate endpoint that summarizes across all the workflows?

Reply:
Ah I was going to document the aggregate endpoint after we fix those metrics. I am happy to include it this pr.